src/serverlist.cpp

```diff
@@ -172,8 +252,9 @@ void CServerListManager::SetCentralServerAddress ( const QString sNCentServAddr
 
     // per definition: If we are in server mode and the directory server address
     // is the localhost address, and set to Custom, we are in directory server mode.
-    bIsCentralServer = ( ( !strCentralServerAddress.toLower().compare ( "localhost" ) || !strCentralServerAddress.compare ( "127.0.0.1" ) ) &&
-                         ( eCentralServerAddressType == AT_CUSTOM ) );
+    bIsCentralServer =
+        ( ( !strCentralServerAddress.compare ( "localhost", Qt::CaseInsensitive ) || !strCentralServerAddress.compare ( "127.0.0.1" ) ) &&
+          ( eCentralServerAddressType == AT_CUSTOM ) );
```

Comment:
Shouldn't this be added as a method somewhere? I mean, it's also repeated in line 275